STBReader/StbData.cs

```diff
@@ -32,37 +32,46 @@ public class StbData
 
         public StbData(string path, double toleLength, double toleAngle)
         {
-            Path = path;
             ToleLength = toleLength;
             ToleAngle = toleAngle;
             
-            var xDocument = XDocument.Load(Path);
-
-            var root = xDocument.Root;
-            if (root != null)
+            try
             {
-                if (root.Attribute("xmlns") != null)
-                {
-                    Xmlns = "{" + (string)root.Attribute("xmlns") + "}";
-                }
-                else
-                {
-                    Xmlns = string.Empty;
-                }
+                XDocument xDocument = XDocument.Load(path);
+                XElement root = xDocument.Root;
 
-                var tmp = (string) root.Attribute("version");
-                switch (tmp.Split('.')[0])
+                if (root != null)
                 {
-                    case "1":
-                        Version = StbVersion.Ver1;
-                        break;
-                    case "2":
-                        Version = StbVersion.Ver2;
-                        break;
+                    if (root.Attribute("xmlns") != null)
+                    {
+                        _xmlns = "{" + (string)root.Attribute("xmlns") + "}";
+                    }
+                    else
+                    {
+                        _xmlns = string.Empty;
+                    }
+
+                    var tmp = (string) root.Attribute("version");
+                    switch (tmp.Split('.')[0])
+                    {
+                        case "1":
+                            _version = StbVersion.Ver1;
+                            break;
+                        case "2":
+                            _version = StbVersion.Ver2;
+                            break;
+                        default:
+                            throw new ArgumentException("The STB version is not set");
+                    }
                 }
+                Init();
+                Load(xDocument);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/hrntsm/STBReader/pullRequest?prid=6384839)